### PR TITLE
Implementation

### PR DESCRIPTION
**Pull request by Gabbendorf**

### CODE REVIEW

File: lib/pascal_triangle.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class PascalTriangle
+
+  def print_lines_to(line_number)
+    all_lines = []
+    lines_numbers = (0..line_number).map {|number| number}
```
Comment (felipesere) [inline lib/pascal_triangle.rb]: Is there any particular reason we need to `map` from number --> number? I wonder if you could do lines 5 and 6 in one go?

Comment (Gabbendorf) [reply lib/pascal_triangle.rb]: You were right! :)

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Pascal's Triangle
```
Comment (RabeaGleissner) [inline README.md:1]: No .gitignore file and you therefore commit our .DS_Store files. Every project should have a .gitignore file. In addition, look into setting a global gitignore file on your laptop where you can always ignore the .DS_Store files.

File: lib/pascal_triangle.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    line = [1]
+    line_number.times {line = next_line(line)}
+    line
+  end
+
+  def next_line(previous_line)
+    new_line = []
+    new_line.push(1)
+    if previous_line.size >= 2
+      (0..previous_line.size-2).each {|index|
```
Comment (RabeaGleissner) [inline lib/pascal_triangle.rb:17]: I think it's uncommon to have this syntax using curly braces and then split it over two lines. If the line gets to long, then use 

[].each do |something|
    something bla bla
end

**Review by RabeaGleissner — COMMENTED** [review]
Sorry, I know that this review is way to late! But I just had the chance to look through your code and thought I'd leave you these comments anyway. 
The code is good, btw. Well done!

**Pull request merged**